packages/shared/src/__tests__/loadClerkJsScript.test.ts

```diff
@@ -0,0 +1,126 @@
+import { buildClerkJsScriptAttributes, clerkJsScriptUrl, loadClerkJsScript } from '../loadClerkJsScript';
+import { loadScript } from '../loadScript';
+
+jest.mock('../loadScript');
+
+describe('loadClerkJsScript(options)', () => {
+  const mockPublishableKey = 'pk_test_Zm9vLWJhci0xMy5jbGVyay5hY2NvdW50cy5kZXYk';
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    (loadScript as jest.Mock).mockResolvedValue(undefined);
+    document.querySelector = jest.fn().mockReturnValue(null);
+  });
+
+  test('throws error when publishableKey is missing', () => {
+    expect(() => loadClerkJsScript({} as any)).rejects.toThrow(
+      'Clerk: Missing publishableKey. You can get your key at https://dashboard.clerk.com/last-active?path=api-keys.',
+    );
+  });
+
+  test('loads script when no existing script is found', async () => {
+    await loadClerkJsScript({ publishableKey: mockPublishableKey });
+
+    expect(loadScript).toHaveBeenCalledWith(
+      expect.stringContaining('https://foo-bar-13.clerk.accounts.dev/npm/@clerk/clerk-js@5/dist/clerk.browser.js'),
```

Comment:
Once we publish a new major this test will fail so we should change `@5` to refer to the package.json version so it updates itself.

Same applies to the URLs below

Reply:
Thanks, updated!